### DISCUSSION TITLE
Can you disable message section of badge?

### DISCUSSION THREAD

**Discussion by Andre601** [badges/shields#6470] [Q&A] [answered]
I would like to only display the label section of a badge alongside a custom SVG icon I add through the `logo=data:` query.

But my question now is, can you hide the message section?
From what I tried so far does it seem like that you can only remove the `label` section through `label=` but not the message.

Is there any way (i.e. through usage of a zero width space) or is that impossible as of now?

**Comment by chris48s** [comment, marked as answer]
Functionally hiding the label or the message do the same thing really, don't they? i.e: if I do
https://img.shields.io/badge/-text-555?logo=javascript
does it matter if `text` is the "label" or the "message"? You've still got a logo and some static text..

**Reply by Andre601** [reply]
I mean, not everyone knows the exact color used by the label section, do they?
